ui/lib/kv/addon/components/kv-data-fields.js

```diff
@@ -23,7 +23,7 @@ import KVObject from 'vault/lib/kv-object';
  * @param {boolean} showJson - boolean passed from parent to hide/shows json editor
  * @param {object} [modelValidations] - object of errors.  If attr.name is in object and has error message display in AlertInline.
  * @param {callback} [pathValidations] - callback function fired for the path input on key up
- * @param {number} [previousVersion] - optional, the version number we're creating a new secret from used to change text for some form labels
+ * @param {boolean} [isEdit=false] - if true, this is a new secret version rather than a new secret. Used to change text for some form labels
```

Comment:
I had made this not a boolean because `@previousVersion` also renders in the old version alert banner. 
<img width="1049" alt="Screenshot 2023-08-16 at 1 32 20 PM" src="https://github.com/hashicorp/vault/assets/68122737/f5d9b499-8e81-4f27-8790-99534e64f099">

Reply:
This is in `kv-data-fields`, `previousVersion` is still on the edit page component. Am I misunderstanding something?

Reply:
Oh! Cool - I forgot I had made this smaller fields only component. Disregard 😄